tests/functions/test_origin_request.py

```diff
@@ -81,7 +94,7 @@ def test_origin_request(
     assert "Item found for '%s'" % real_uri in caplog.text
 
     assert request == {
-        "uri": "/e4a3f2sum",
+        "uri": "/e4a3f2sum&ResponseContentType={}".format(content_type),
```

Comment:
I think it's showing the wrong behavior:

- should be ? instead of &, right?
- the content types should be url-encoded